### DISCUSSION TITLE
Icons diagram

### DISCUSSION THREAD

**Discussion by nirname** [mermaid-js/mermaid#4583] [Ideas]
There are many tools that allow to customize Readme with icons. I know that mermaid is not about that, but it is supported by many organizations, and it would be very convenient to embed icons list:

```mermaid-example
icons centered (left/right/justified) perline=5

js,ts,cpp,oracle,git,linux,mermaid,dot,seqdiag,gnuplot
```

**Comment by knsv** [comment]
I can see how this would be convenient for many diagram types. Mot important is that this is managed in a way so it can be consistent between the different diagrams.

I was thinking on for instance having icons as actors in sequence diagrams or as a node in a flowchart

**Comment by nirname** [comment]
So icons could be a list of nodes on top of which a layout algorithm is applied
